bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -459,8 +460,8 @@ public void run() {
                                 numReqInLastForceWrite = 0;
                             }
                         }
+                        numReqInLastForceWrite += req.process(shouldForceWrite);
```

Comment:
I think the changes in this file are not needed. probably come from merge?

Reply:
will change it, Seems it is from merge.